settings.py

```diff
@@ -44,27 +58,44 @@
 LOGGING['root']['level'] = env('DJANGO_LOG_LEVEL', 'INFO')
 
 AWS_STORAGE_BUCKET_NAME = env('AWS_STORAGE_BUCKET_NAME', '')
+AWS_BUCKET_DOMAIN = env('AWS_BUCKET_DOMAIN', AWS_STORAGE_BUCKET_NAME + '.s3.amazonaws.com')
```

Comment:
I don't think this is actually used anywhere, is it?

Reply:
Right this is a bit of a hack, if you're not using S3 for media but normal Django storage you need to specify the domain + media url prefix (e.g. `rapidpro.myhost.com/media`) here because all throughout the RP code base `settings.AWS_BUCKET_DOMAIN` is used to construct the URL to statics.

Reply:
👍